CollectionsExcemptionsHomework/.idea/compiler.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
please add .idea files to .gitignore

Reply:
Nu stiu cum sa fac chestia asta si sigur imi v-a fi de ajutor ca tot timpul am probleme cu idea files cand incerc sa switch branches

Reply:
https://stackoverflow.com/questions/9550437/how-to-make-git-ignore-idea-files-created-by-rubymine